lib/debug/session.rb

```diff
@@ -481,6 +481,17 @@ def process_command line
           return :retry
         end
 
+      # * `ls`
+      #   * Executes `irb`'s `ls` command with current binding.
+      # * `ls <obj>`
+      #   * Executes `irb`'s `ls` command with the given obj.
+      when 'ls'
+        if @ui.remote?
+          @ui.puts "not supported on the remote console."
```

Comment:
For some reason it can't run on remote console either. I think we can sort it out later.